generators/common/types.d.ts

```diff
@@ -1,17 +1,28 @@
 import type { HandleCommandTypes } from '../../lib/command/types.js';
 import type {
-  Application as BaseApplicationApplication,
   Config as BaseApplicationConfig,
   Entity as BaseApplicationEntity,
-  Field as BaseApplicationField,
   Options as BaseApplicationOptions,
-  Relationship as BaseApplicationRelationship,
   Source as BaseApplicationSource,
 } from '../base-application/types.js';
+import type { Application as DockerApplication } from '../docker/types.js';
```

Comment:
Docker is common?

Reply:
Yes, every client-server application will have docker dbs and other utilities.